<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1881</width>
      <height>961</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTabWidget" name="tabWidget">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QTabBar::tab {
	background: #72CCEE;
	color: #003042;
	border: 1px solid #0C65AF;
	padding: 6px;
	border-top-right-radius: 8px;
	border-top-left-radius: 8px;
}

QTabBar::tab:selected {
	background: #FFFFFF;
	border: 1px solid #0C65AF;
    border-top-right-radius: 8px;
	border-top-left-radius: 8px;
	border-bottom: none;
}
</string>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="tab_3">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QTabBar::tab {
	background: #72CCEE;
	color: #003042;
	border: 1px solid #0C65AF;
	padding: 6px;
	border-top-right-radius: 8px;
	border-top-left-radius: 8px;
}

QTabBar::tab:selected {
	background: #FFFFFF;
	border: 1px solid #0C65AF;
    border-top-right-radius: 8px;
	border-top-left-radius: 8px;
	border-bottom: none;
}
</string>
        </property>
        <attribute name="title">
         <string>Автомобили</string>
        </attribute>
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>1851</width>
           <height>901</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QTableView" name="carTableView">
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>14</pointsize>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
            </property>
            <property name="dragEnabled">
             <bool>false</bool>
            </property>
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderMinimumSectionSize">
             <number>200</number>
            </attribute>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>369</number>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="addCarButton">
              <property name="text">
               <string>Добавить</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="editCarButton">
              <property name="text">
               <string>Редактировать</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="deleteCarButton">
              <property name="text">
               <string>Удалить</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QTabBar::tab {
	background: #72CCEE;
	color: #003042;
	border: 1px solid #0C65AF;
	padding: 6px;
	border-top-right-radius: 8px;
	border-top-left-radius: 8px;
}

QTabBar::tab:selected {
	background: #FFFFFF;
	border: 1px solid #0C65AF;
    border-top-right-radius: 8px;
	border-top-left-radius: 8px;
	border-bottom: none;
}
</string>
        </property>
        <attribute name="title">
         <string>Клиетны</string>
        </attribute>
        <widget class="QWidget" name="verticalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>1851</width>
           <height>901</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QTableView" name="clientTableView">
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>14</pointsize>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
            </property>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>369</number>
            </attribute>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="addClientButton">
              <property name="text">
               <string>Добавить</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="editClientButton">
              <property name="text">
               <string>Редактировать</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="deleteClientButton">
              <property name="text">
               <string>Удалить</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_4">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QTabBar::tab {
	background: #72CCEE;
	color: #003042;
	border: 1px solid #0C65AF;
	padding: 6px;
	border-top-right-radius: 8px;
	border-top-left-radius: 8px;
}

QTabBar::tab:selected {
	background: #FFFFFF;
	border: 1px solid #0C65AF;
    border-top-right-radius: 8px;
	border-top-left-radius: 8px;
	border-bottom: none;
}
</string>
        </property>
        <attribute name="title">
         <string>Аренда</string>
        </attribute>
        <widget class="QWidget" name="verticalLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>1851</width>
           <height>901</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QTableView" name="rentalsTableView">
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>14</pointsize>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
            </property>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>369</number>
            </attribute>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="1" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Активных аренд:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="rentOutCarButton">
              <property name="text">
               <string>Выдать в аренду</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="editRentButton">
              <property name="text">
               <string>Редактировать запись</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="returnFromRentalButton">
              <property name="text">
               <string>Удалить</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_5">
        <attribute name="title">
         <string>Страница</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_6">
        <attribute name="title">
         <string>Страница</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
